docs/source/howto/level1-csd.rst

```diff
@@ -0,0 +1,312 @@
+      If in a file, provide one coordinate per line.
+      If on the command line, provide a comma separated list.
+  (2) a shapefile (point/polygon/line). On-the-fly reprojection is provided,
+      but using EPSG4326 is recommended.
+  (3) grid identifier:
+      Landsat: Path/Row as "PPPRRR". Make sure to keep leading zeros:
+        correct: 181034, incorrect: 18134
+      Sentinel-2: MGRS tile as "TXXXXX". Make sure to keep the leading T before
+        the MGRS tile number.
+      You can either give the path to a file, or give the IDs on the command line.
+      If in a file, provide one ID per line.
+      If on the command line, provide a comma separated list.
+
+The first three arguments specify the local directories for the metadata catalogues, the level 1 datapool (download directory), and the processing queue. We define our area of interest with the fourth argument. FORCE Level 1 CSD offers quite some flexibility in how the aoi can be specified. Options ``(1)`` and ``(2)`` will intersect the provided geometries with the tiling scheme of Landsat and/or Sentinel-2, while option ``(3)`` the third option can be used if we already know which Landsat path/row or Sentinel-2 tiles we need. If you want to specify coordinates or a list of grid identifiers on the command line, keep in mind that the maximum length of command line calls is limited and varies by system! So your lists may just be cut off and in the worst case you won't notice before it's too late. You may be better off storing them in a text file. Make sure this file uses UNIX end of line characters and provide one coordinate pair or grid identifier per line.
+
+Optional arguments
+"""""""""""""""""""
+Four of the optional arguments can be used to narrow down our search results:
+
+.. code-block:: none
+
+  -c | --cloudcover
+  minimum,maximum
+  The cloud cover range must be specified in %
+  Default: 0,100
+
+  -d | --daterange
+  starttime,endtime
+  Dates must be given in the following format: YYYYMMDD,YYYYMMDD
+  Default: 19700101,today
+
+  -s | --sensor
+  Sensors to include in the query, comma-separated.
+  Valid sensors:
+  Landsat                             Sentinel
+  LT04 - Landsat 4 TM                 S2A - Sentinel-2A MSI
+  LT05 - Landsat 5 TM                 S2B - Sentinel-2B MSI
+  LE07 - Landsat 7 ETM+
+  LC08 - Landsat 8 OLI
+  Default: LT04,LT05,LE07,LC08,S2A,S2B
+
+  -t | --tier
+  Landsat collection tier level. Valid tiers: T1,T2,RT
+  Default: T1
+
+We can filter by cloud cover estimate, date range, sensor, and tier (Landsat only). If these parameters aren't set, they will default to the values above. Note that the default search is a very broad one. It will basically return all Landsat **and** Sentinel-2 scenes ever acquired for the area of interest. The only exception is the Tier level for Landsat - only Tier 1 scenes will be included, as Tier 2 and Real Time scenes are not desirable for most applications.
```

Comment:
This here is very important. I suggest to put everything after Note into a .. caution:: .. danger:: or .. attention:: box

Reply:
hmm, not sure if I see the correct version? I see a thumbs up, but no change